.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
+  "globals": {
```

Comment:
There are a *ton* of globals defined in edx-platform's .jshintrc:

https://github.com/edx/edx-platform/blob/master/.jshintrc#L111

Are you thinking that we'll import this eslintrc file and then extend it again with platform specific settings? Can it chain multiple times like this?

At the very least, I think we need to add the Jasmine globals (unless AirBnB defines them already).